schema-engine/connectors/sql-schema-connector/src/flavour/mysql/connection.rs

```diff
@@ -115,6 +115,30 @@ impl Connection {
         self.0.query_raw(sql, params).await.map_err(quaint_err(url))
     }
 
+    pub(super) async fn parse_raw_query(
+        &self,
+        sql: &str,
+        url: &MysqlUrl,
+        circumstances: BitFlags<super::Circumstances>,
+    ) -> ConnectorResult<quaint::connector::ParsedRawQuery> {
+        tracing::debug!(query_type = "parse_raw_query", sql);
+        let mut parsed = self.0.parse_raw_query(sql).await.map_err(quaint_err(url))?;
+
+        if circumstances.contains(super::Circumstances::IsMysql56)
+            || circumstances.contains(super::Circumstances::IsMysql57)
+        {
+            parsed.parameters = parsed
+                .parameters
+                .into_iter()
+                .map(|p| p.set_typ(ColumnType::Unknown))
+                .collect();
+
+            Ok(parsed)
+        } else {
+            Ok(parsed)
+        }
+    }
```

Comment:
I'm forcing MySQL 5.6 & 5.7 to have params as `Unknown` instead of `Bytes` which is unhelpful and misleading.